products/rhel9/profiles/ospp.profile

```diff
@@ -11,14 +11,8 @@ reference: https://www.niap-ccevs.org/Profile/PP.cfm
 title: '[DRAFT] Protection Profile for General Purpose Operating Systems'
 
 description: |-
-    This profile reflects mandatory configuration controls identified in the
-    NIAP Configuration Annex to the Protection Profile for General Purpose
-    Operating Systems (Protection Profile Version 4.2.1).
-
-    This configuration profile is consistent with CNSSI-1253, which requires
-    U.S. National Security Systems to adhere to certain configuration
-    parameters. Accordingly, this configuration profile is suitable for
-    use in U.S. National Security Systems.
+    This profile is part of Red Hat Enterprise Linux 9 Common Criteria Guidance documentation for Target of Evaluation based on Protection Profile for General Purpose Operating Systems (OSPP) version 4.2.1 and Functional Package for SSH version 1.0.
```

Comment:
I wonder where this OpenSSH 1.0 version comes from.

Reply:
It's not OpenSSH.

It's Functional Package for SSH in https://www.niap-ccevs.org/Profile/Info.cfm?PPID=459&id=459 which replaced the older Extended Package (https://www.niap-ccevs.org/Profile/Info.cfm?PPID=389&id=389).

Reply:
Thanks for the explanation.